panel/widgets/tables.py

```diff
@@ -633,7 +633,7 @@ class DataFrame(BaseTable):
     hierarchical = param.Boolean(default=False, constant=True, doc="""
         Whether to generate a hierachical index.""")
 
-    fit_columns = param.Boolean(default=None, doc="""
```

Comment:
This seems unrelated.

Reply:
The changes i made gave two type of errors, though both related to `controls(jslink=True)`.  One of the error was related to `pn.indicators.Progress` and the other `pn.widgets.DataFrame`. The one for DataFrame can be reproduced by:
```python
df_widget = pn.widgets.DataFrame()
df_widget.controls(jslink=True)
# ValueError: Boolean 'value' only takes a Boolean value.
```

I could see in the [documentation](https://panel.holoviz.org/reference/widgets/DataFrame.html) that it should be True by default.